### PR TITLE
Create your own Enumerable

### PR DESCRIPTION
**Pull request by denisdiaconu**
In this project we used a module inside our class. Our module MyEnumerable is a subset of the functionality of Enumerable.

### CODE REVIEW

File: index.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*enumerables)
+    @enumerables = enumerables
+  end
+end
```
Comment (DeliceLydia) [inline index.rb]: - [ ] `MyList` class should implement `#each` method that yields all members of `@list` in succession please work on that.. you can read about how to use [yield](https://www.educba.com/yield-in-ruby/) in ruby

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.rb [hunk 2]
```diff
@@ -5,6 +5,10 @@ class MyList
   def initialize(*enumerables)
     @enumerables = enumerables
   end
+
+  def each(&block)
+    @enumerables.each(&block)
+  end
 end
```
Comment (DeliceLydia) [inline index.rb]: - [ ] `Each` method should `yield` all members of `@enumerables` in succession
Remember that we use `yield` when we want to call any method with the block code

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Hi Team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._



### Don't give up!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ElijahTobs — APPROVED** [review]
# STATUS: APPROVED 🎉 🥇 
## Hi @denisdiaconu and @SimonGrchevski , 

Good job making the changes required by the previous Code Reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

_Reviewed by_: __**Elijah Ayandokun**__

**Pull request merged**